eventq_aws/lib/eventq_aws/aws_queue_manager.rb

```diff
@@ -8,7 +8,7 @@ class QueueManager
       def initialize(options)
 
         if options[:client] == nil
-          raise ':client (QueueClient) must be specified.'.freeze
+          EventQ.log(:info, "[#{self.class}] - options[:client] is now deprecated!!, please pass options[:mq_endpoint].")
```

Comment:
the comment here in the log message is about `mq_endpoint` which is for rabbitmq and not aws.  But this is the aws worker.